src/components/AddPlaylist/AddPlaylist.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import AddPlaylist from "./AddPlaylist";
+import { ThemeProvider } from "styled-components";
+import theme from "../../theme";
+import userEvent from "@testing-library/user-event";
+
+const mockedDispatch = jest.fn();
+jest.mock("react-redux", () => ({
+    useDispatch: () => mockedDispatch,
+}));
+
+describe("AddPlaylist component", () => {
+    it("renders without crashing", () => {
+        render(
+            <ThemeProvider theme={theme}>
+                <AddPlaylist />{" "}
+            </ThemeProvider>
+        );
+    });
+    it("shows helper text when given playlist name is too long", async () => {
+        const { getByTestId } = render(
+            <ThemeProvider theme={theme}>
+                <AddPlaylist />{" "}
+            </ThemeProvider>
+        );
+
+        const input = getByTestId("name-input");
+        await userEvent.type(input, "Looooooooooooooooooooooooooooooooooooooooooooooooooonnnng playlist name");
+
+        const helperText = getByTestId("helper-text");
+        expect(helperText.textContent).toEqual("Too long name for the playlist!");
+    });
+    it("shows helper text when given playlist name is too long", async () => {
```

Comment:
the test names are wrong in some instances

Reply:
fixed